Только в /root/dk_scripts/tmp/1: 4-Dec-2011183802LINK-I-Upfa13KTCS2-SW1-48_192.168.22.84.conf
diff -r /root/dk_scripts/tmp/2/TBR21-SW1-48_192.168.22.70.conf /root/dk_scripts/tmp/1/TBR21-SW1-48_192.168.22.70.conf
381a382
>  shutdown
Только в /root/dk_scripts/tmp/2: TCS2-SW1-48_192.168.22.84.conf
